net/unbound/Makefile

```diff
@@ -37,9 +37,9 @@ define Package/unbound
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=IP Addresses and Names
+  USERID:=unbound=553:unbound=553
```

Comment:
Do you mean an installation where unbound is part of the base image? IIRC `USERID` is handled by `opkg`.